data/compatible_products.yml

```diff
@@ -77,7 +77,7 @@ sets:
           image_width: 195
           download: 'https://www.payara.fish/downloads/payara-platform-community-edition/'
           versions:
-            - version: 6.2022.1
+            - version: 6.2022.1, Web Profile
```

Comment:
Isn't the full server also web profile compatible?

Reply:
No, Payara full server being added to web profile wasn't part of the issue. 

Also check: https://docs.payara.fish/community/docs/6.2022.1/Jakarta%20EE%20Certification/6.2022.1/6.2022.1%20Web%20TCK%20Results.html

The sidebar has all the certs. The full server for web profile isn't listed.